src/traces/pie/plot.js

```diff
@@ -559,18 +559,31 @@ function prerenderTitles(cdModule, gd) {
 }
 
 function transformInsideText(textBB, pt, cd0) {
-    var textDiameter = Math.sqrt(textBB.width * textBB.width + textBB.height * textBB.height);
+    var r = cd0.r || pt.rpx1;
+    var rInscribed = pt.rInscribed;
+
+    var isEmpty = pt.startangle === pt.stopangle;
+    if(isEmpty) {
+        return {
+            rCenter: 1 - rInscribed,
+            scale: 0,
```

Comment:
So, by _fitting_ inside text of zero-value sunburst sectors, you mean setting their `scale` transform to `0`, correct?

Reply:
Correct.

Reply:
The other option (not implemented here) may be not to draw sectors with zero values.

Reply:
Or set them with `display: 'none'`, to keep the sector <--> node map 1:1

Reply:
... but really, this is fine. It allows us to keep all the logic in `transformInsideText`.